acq400_hapi/acq400.py

```diff
@@ -902,7 +902,7 @@ class Acq2106(Acq400):
     Defines features specific to ACQ2106
     """
 
-    def __init__(self, _uut, monitor=True, has_mgtdram=False):
+    def __init__(self, _uut, monitor=True, has_mgtdram=False, is_bolo=False):
```

Comment:
let's make it "has_bolo" please.  This _is_ an ACQ2106, it might also have mgtdram or bolo.

I'm not sure If I even like this. "has_dsp" would be the best switch...